weather/test/handle_weather_data_tests.py

```diff
@@ -0,0 +1,20 @@
+from weather.core.handle_weather_data import write_content_to_json, get_webpage_content_in_json_format
+import unittest
+import os
+
+class MyTestCase(unittest.TestCase):
+    def setUp(self):
+        self.mock_dict = {"FeelsLikeC": "5",
```

Comment:
Ezzel az appod melyik részét mokkolod? 
"Mock testing is an approach to unit testing that lets you make assertions about how the code under test is interacting with other system modules."
Ez itt nem reprezentálja egyik részét se a rendszerednek. (Nem a JsonData nem így néz ki.)